admin/app/helpers/spree/admin/bulk_operations_helper.rb

```
@@ -0,0 +1,60 @@
+
+        if options[:icon]
+          text = icon(options[:icon]) + ' ' + text
+        end
+
+        content_tag :span, data: { toggle: 'modal', target: '#bulk-modal' } do
+          link_to text, path, options
+        end
+      end
+
+      def bulk_operations_close_button
+        button_tag('', type: 'button', class: 'btn-close', data: { dismiss: 'modal', aria_label: Spree.t(:close), action: 'bulk-operation#cancel' }).html_safe
```

RuboCop (1.73)

[convention] Line is too long. [158/150]

(Layout/LineLength)